Remove top adm unit types

bszady opened the pull request:
1) Removal of top-level administrative unit types represented by 'isSymboliseByMeansOf' map_symbol_100. Except for: administrative_type_57, 135, 143. 
2) Example of label change for administrative unit types (111, 112, 113) that are represented by 'administrative_system_19'.

mereolog commented on ontohgis.ttl:
```diff
@@ -5311,8 +5123,8 @@ ontohgis:administrative_type_111 rdf:type owl:Class ;
-                                            "powiat"@pl ,
+                                            "district (Silesia (1372-1782))"@en ,
+                                            "powiat (Śląsk (1372-1782))"@pl ,
                                             "terra"@la ;
```
Why wasn't the Latin label updated alongside the others?

mereolog replied:
> We have no Latin names for administrative system, at this moment. The same is with label @de. What do you suggest?

http://purl.org/ontohgis#administrative_type_111 has the Latin label "terra"@la.

bszady replied:
Yes, but we have no Latin term for the administrative system which is going to be a part of label for administrative type. We have mostly pl and en labels for administrative systems

mereolog reviewed (commented):
There is nothing particularly wrong with this review.
Still I would like to see the top-level administrative units to return as individuals - without them one the most substantial distinctions in this ontology will disappear.

bszady commented:
> There is nothing particularly wrong with this review. Still I would like to see the top-level administrative units to return as individuals - without them one the most substantial distinctions in this ontology will disappear.

I have tried to add an indivudual for Holy Roman Empire. I have made the commit, but I can't to make a push. I have a warning: $ git push
To https://github.com/ontohgis/ontology.git
 ! [rejected]        remove_top_adm_unit_types -> remove_top_adm_unit_types (non-fast-forward)
error: failed to push some refs to 'https://github.com/ontohgis/ontology.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

bszady commented:
I can't uderstand why I have an info:
$ git status
On branch remove_top_adm_unit_types
Your branch and 'origin/remove_top_adm_unit_types' have diverged,
and have 1 and 1 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)
nothing to commit, working tree clean

bszady commented:
I have added one commit with top-level administrative system as individual (no 49). And I added a dcterms::isPartOf between administrative_system_7 and administrative_system_49.

mereolog commented on ontohgis.ttl:
```diff
@@ -17236,6 +16854,21 @@ ontohgis:administrative_system_48 rdf:type owl:NamedIndividual ,
+ontohgis:administrative_system_49 rdf:type owl:NamedIndividual ,
+                                           ontohgis:object_21 ;
+                                  ontohgis:endsDominationAt "1806-08-06"^^xsd:date ;
+                                  ontohgis:hasBeginning "800-12-25"^^xsd:date ;
```
Pellet reasoner throws the following error:
An error occurred during reasoning: Cannot do reasoning with inconsistent ontologies!
Reason for inconsistency: Literal value "800-12-25"^^date is not valid for the rdatatype date.
org.mindswap.pellet.exceptions.InconsistentOntologyException: Cannot do reasoning with inconsistent ontologies!
Reason for inconsistency: Literal value "800-12-25"^^date is not valid for the rdatatype date

I think the value should be "0800-12-25"

bszady replied:
ok

mereolog commented on ontohgis.ttl:
```diff
@@ -17236,6 +16854,21 @@ ontohgis:administrative_system_48 rdf:type owl:NamedIndividual ,
+                                  ontohgis:hasEnd "1806-08-06"^^xsd:date ;
+                                  ontohgis:hasScope "terytorium wielokrotnie zmieniało swój zasięg, obejmowało głównie kraje Europy Środkowej i Wschodniej, w przeważającej części zamieszkiwane przez ludność posługującą się językiem niemieckim"@pl ;
+                                  ontohgis:individualHasExternalIdentifier 49 ;
+                                  ontohgis:startsDominationAt "962-02-02"^^xsd:date ;
```
The value here should read "0962-02-02".

bszady replied:
Ok

mereolog commented on ontohgis.ttl:
```diff
@@ -17282,6 +16915,7 @@ ontohgis:administrative_system_7 rdf:type owl:NamedIndividual ,
                                  ontohgis:individualHasExternalIdentifier 7 ;
                                  ontohgis:startsDominationAt "1815-01-01"^^xsd:date ;
                                  dc:creator ontohgis:person_11 ;
+                                 <http://purl.org/dc/terms/isPartOf> ontohgis:administrative_system_49 ;
```
I would use P46i_forms_part_of instead of http://purl.org/dc/terms/isPartOf because the latter is an annotation property and we need an object property here.

bszady replied:
Ok. Agree
